frontend/src/pages/detail/Detail.tsx

```diff
@@ -124,43 +124,39 @@ const S_MentorInfoWrapper = styled.div`
 
 const S_TapWrapper = styled.div`
   display: flex;
-  flex-direction: row;
-  position: relative;
-
   width: 100%;
-  padding: 1rem;
+  overflow: hidden;
 `;
 
-const S_Tap = styled.p<{ selected: boolean }>`
-  width: 50%;
+const S_Tap = styled.div<{ selected: boolean }>`
+  flex: 1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
   cursor: pointer;
-
-  text-align: center;
-
-  ${({ theme }) => theme.TYPOGRAPHY.B2_B};
-`;
-
-const S_TapIndicator = styled.div<{ selected: 'detail' | 'review' }>`
-  position: absolute;
-  bottom: 0;
-  left: 0;
-  z-index: 0;
-
-  width: 50%;
-  height: 1px;
-
-  background-color: ${({ theme }) => theme.SYSTEM.MAIN500};
-  transition: transform 0.2s ease-in-out;
-
-  transform: ${({ selected }) =>
-    selected === 'detail' ? 'translateX(0%)' : 'translateX(100%)'};
+  padding: 1.6rem 0;
+  border-top: 1px solid
+    ${({ selected, theme }) => (selected ? 'none' : theme.SYSTEM.GRAY50)};
+  border-bottom: 1px solid
+    ${({ selected, theme }) => (selected ? 'none' : theme.SYSTEM.GRAY50)};
+  background-color: ${({ selected, theme }) =>
+    selected ? theme.SYSTEM.GRAY800 : theme.BG.WHITE};
+  color: ${({ selected, theme }) =>
+    selected ? theme.BG.WHITE : theme.FONT.B01};
+
+  transition:
+    background-color 0.25s ease,
+    color 0.25s ease;
+
+  ${({ theme }) => theme.TYPOGRAPHY.B2_R};
 `;
 
 const S_ContentWrapper = styled.div`
   display: flex;
 
   width: 100%;
-  padding-top: 2rem;
+  padding: 0 2rem;
+  padding-top: 4rem;
```

Comment:
p4: 이렇게 줄일 수 있을 것 같아 😊

```suggestion
  padding: 4rem 2rem 0;
```

---

padding top의 값이 4rem보다 작은 건 어때?

<img width="297" height="200" alt="스크린샷 2025-09-19 오후 6 05 53" src="https://github.com/user-attachments/assets/13c4f415-6c2a-4714-9f4f-799dfefcd13a" />

Reply:
p4: 멘토 소개가 위의 김트레이너와 padding이 일치하는 점 너무 좋은 것 같아 👍

자격 사항도 같이 일치하면 더 좋을 것 같은데 어떻게 생각해? 🧐
<img width="297" height="401" alt="스크린샷 2025-09-19 오후 6 07 55" src="https://github.com/user-attachments/assets/771626e9-1647-48f1-826e-afc3491c5bf0" />

Reply:
padding이 같은게 훨씬깔끔해 보여서 그렇게 변경할게!

Reply:
[style: padding 축약형으로 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/693/commits/b99af528099bf1c12c8bc28ceb359f0701067da0) 

[style: 페이지의 padding이 같게 상세페이지 컨텐츠 padding 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/693/commits/cc47f9f7b81ebba32da932c1e5e90f7241c73684) 

[style: 디페일 페이지 컨텐츠 padding-top 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/693/commits/fce877f7d9d38006768248d5f1107f72de889256)